### PR TITLE
Ria 18843

### PR DESCRIPTION
**Pull request by steleug**
https://smartbear.atlassian.net/browse/RIA-18843

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <requireMavenVersion-value>3.0.5</requireMavenVersion-value>
-        <maxJdkVersion-value>12</maxJdkVersion-value>
+        <maxJdkVersion-value>16</maxJdkVersion-value>
```
Comment (steleug) [inline pom.xml:30]: Change is here.

File: pom.xml [hunk 2]
```diff
@@ -43,7 +43,7 @@
         <groovy-all.version>3.0.6</groovy-all.version>
 
         <!-- org.apache.maven.plugins -->
-        <maven-enforcer-plugin-version>3.0.0-M2</maven-enforcer-plugin-version>
+        <maven-enforcer-plugin-version>3.0.0-M3</maven-enforcer-plugin-version>
```
Comment (steleug) [inline pom.xml]: Change is here.

File: pom.xml [hunk 3]
```diff
@@ -64,7 +64,7 @@
         <exec-maven-plugin-version>1.6.0</exec-maven-plugin-version>
         <xmlbeans-maven-plugin-version>2.3.5-sb-fixed</xmlbeans-maven-plugin-version>
         <!-- org.openjfx-->
-        <javafx.version>15</javafx.version>
+        <javafx.version>16</javafx.version>
```
Comment (steleug) [inline pom.xml:67]: Change is here.

File: pom.xml [hunk 4]
```diff
@@ -82,7 +82,62 @@
         <xerces-xercesImpl-version>2.12.0</xerces-xercesImpl-version>
         <commons-beanutils-version>1.9.4</commons-beanutils-version>
         <log4j2.version>2.17.0</log4j2.version>
-
+        <mockito-all.version>1.9.5</mockito-all.version>
```
Comment (steleug) [inline pom.xml:85]: Change is here.

File: pom.xml [hunk 5]
```diff
@@ -82,7 +82,62 @@
...
+        <extra-enforcer-rules.version>1.3</extra-enforcer-rules.version>
```
Comment (steleug) [inline pom.xml:86]: Change is here.

**Review by steleug — COMMENTED** [review]
Main changes were made in marked places.

File: pom.xml (hunk 5 above)
Comment (AlexanderYeremeyev) [inline pom.xml:86]: Why we need this version changes. Previosly it was 1.1

File: pom.xml (hunk 2 above)
Comment (AlexanderYeremeyev) [reply pom.xml]: Why we need it?

File: pom.xml [hunk 6]
```diff
@@ -82,7 +82,62 @@
...
+        <hamcrest-all.version>1.3</hamcrest-all.version>
+        <jsr173_api.version>1.0</jsr173_api.version>
+        <com.google.code.findbugs.version>2.0.0</com.google.code.findbugs.version>
+        <jsr305.version>2.0.0</jsr305.version>
```
Comment (AlexanderYeremeyev) [inline pom.xml]: This version is absent. Please describe?

**Review by AlexanderYeremeyev — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 6 above)
Comment (steleug) [reply pom.xml]: It was refactoring, use com.google.code.findbugs.version instead this now. Nice catch, how did you do this? mvn dependency:analyze shows too many information and I skipped it.

File: pom.xml (hunk 5 above)
Comment (steleug) [reply pom.xml:86]: axm use 1.3 version, it allows set 16 java to rule

File: pom.xml (hunk 2 above)
Comment (steleug) [reply pom.xml]: For consistent with axm maven-enforcer-plugin-version. Nevermind, I've reverted back it.

**Review by AlexanderYeremeyev — APPROVED** [review]

**Pull request merged**